### DISCUSSION TITLE
Russian Locale encodeURI

### DISCUSSION THREAD

**Discussion by zielinskiartsite** [nuxt-modules/i18n#1353] [Q&A] [answered]
Hi i have a small problem with Russian locale. I use module version: 7.2.0

In Component Configuration:

` nuxtI18n: {
    paths: {
      en: '/blog',
      pl: '/blog',
      ru: '/блог',
    },`

It cause the error in console, and 404 when link to this page (/ru/блог): 

`[vue-router] Route with path "/ru/блог" contains unencoded characters, make sure your path is correctly encoded before passing it to the router. Use encodeURI to encode static segments of your path.`

Do you have an idea how to solve it?

**Comment by rchl** [comment, marked as answer]
Try `encodeURI` on the russian path as it says.

**Reply by zielinskiartsite** [reply]
oh, it was so easy, thanks a lot

` nuxtI18n: {
    paths: {
      en: '/blog',
      pl: '/blog',
      ru: encodeURI('/блог'),
    },
  },`
